triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -243,7 +237,7 @@ object Server {
                                 ctx,
                                 token,
                                 trigger,
-                                params,
+                                params.identifier,
```

Comment:
I think this is the trigger script identifier? If we change the name of `triggerOrigId` it would be nice to change this to be consistent (or even if we don't).